### PR TITLE
Allows to differentiate between timings...

### PR DESCRIPTION
**Pull request by fhaScireum**
...which should be shown to all users and which should be shown to administrators only

- Fixes: SIRI-250

### CODE REVIEW

**Review by qw3ry — APPROVED** [review]

File: src/main/java/sirius/kernel/async/BasicTaskContextAdapter.java [hunk 1]
```diff
@@ -61,6 +61,11 @@ public void addTiming(String counter, long millis) {
         // Ignored by the default implementation.
```
Comment (andyHa) [inline src/main/java/sirius/kernel/async/BasicTaskContextAdapter.java:61]: provide a default impl here

**Review by sabieber — APPROVED** [review]

File: src/main/java/sirius/kernel/async/BasicTaskContextAdapter.java (hunk 1 above)
Comment (andyHa) [reply src/main/java/sirius/kernel/async/BasicTaskContextAdapter.java:61]: or rather kill, das this method is never invoked?

File: src/main/java/sirius/kernel/async/TaskContext.java [hunk 2]
```diff
@@ -167,6 +167,22 @@ public void addTiming(String counter, long millis) {
         adapter.addTiming(counter, millis);
```
Comment (andyHa) [inline src/main/java/sirius/kernel/async/TaskContext.java:167]: call  adapter.addTiming(counter, millis, false);

File: src/main/java/sirius/kernel/async/TaskContextAdapter.java [hunk 3]
```diff
@@ -61,6 +61,15 @@
      */
     void addTiming(String counter, long millis);
```
Comment (andyHa) [inline src/main/java/sirius/kernel/async/TaskContextAdapter.java:62]: kill?

**Review by andyHa — CHANGES_REQUESTED** [review]
...

**Review by idlira — APPROVED** [review]

**Review by andyHa — APPROVED** [review]

**Review by mkeckmkeck — APPROVED** [review]

**Pull request merged**